internal/types/document.go

```diff
@@ -184,9 +220,9 @@ func (d *Document) Command() string {
 func (d *Document) add(key string, value any) error {
```

Comment:
It doesn't error in this func, any reason we want to keep error return value?

Reply:
We already have a tracking issue for refactoring of bson and related packages